tests/servers/math_server.py

```diff
@@ -16,13 +16,11 @@ def multiply(a: int, b: int) -> int:
 
 
 @mcp.prompt()
-def configure_assistant(skills: str) -> str:
-    return [
-        {
+def configure_assistant(skills: str) -> dict:
```

Comment:
the list was here on purpose actually (to test lists of messages returned). but the type annotation is actually incorrect :)

Reply:
thanks